src/main/kotlin/io/github/cottonmc/jsonfactory/builder/JsonFactoryBuilder.kt

```diff
@@ -0,0 +1,96 @@
+package io.github.cottonmc.jsonfactory.builder
+
+import io.github.cottonmc.jsonfactory.data.Identifier
+import io.github.cottonmc.jsonfactory.frontend.Frontend
+import io.github.cottonmc.jsonfactory.frontend.Generator
+import io.github.cottonmc.jsonfactory.frontend.MessageType
+import io.github.cottonmc.jsonfactory.gens.ContentGenerator
+import java.io.File
+
+/**
+ * Builder class, as an API frontend.
+ * */
+class JsonFactoryBuilder() : Frontend {
```

Comment:
The parentheses in the constructor aren't needed.

Reply:
removed.